whiskey.yml

```diff
@@ -44,55 +32,44 @@ Build:
     OnlyDuring: Clean
     Path: reset.ps1
 
-- GetPowerShellModule:
-    Name: Carbon.Core
-    Version: 1.0.0
-    Path: Carbon.Cryptography\Modules
-
-- Pester4:
-    Script: Tests\*.Tests.ps1
-
-# Build runs twice on some servers, so delete any ZIP file created by another build.
 - Delete:
-    Path: .output\*.zip
+    OnlyBy: BuildServer
+    Path: Carbon.Cryptography\prism.*
+
+- CopyFile:
+    Path:
+    - CHANGELOG.md
+    - README.md
+    - LICENSE
+    - NOTICE
+    DestinationDirectory: Carbon.Cryptography
 
 - Zip:
-    ArchivePath: .output\Carbon.Cryptography.zip
+    ArchivePath: .output\Carbon.Cryptography-$(WHISKEY_SEMVER2).zip
     Path: 
     - Carbon.Cryptography
+    Exclude:
+    - "*.pdb"
+    - "*.orig"
+    - "prism.*"
 
-Publish:
-
-- AppVeyorWaitForBuildJobs:
-    IfExists: env:PUBLISH
-    UnlessExists: env:APPVEYOR_PULL_REQUEST_NUMBER
-    ApiKeyID: AppVeyorBearerToken
+- Exec:
+    OnlyBy: BuildServer
+    Path: appveyor
+    Argument: [ 'PushArtifact', '.output/Carbon.Cryptography-$(WHISKEY_SEMVER2).zip', '-DeploymentName', 'GitHub' ]
 
 - PublishPowerShellModule:

```

Comment:
I think this task is supposed to be removed if we're publishing via AppVeyor?

Reply:
AppVeyor doesn’t natively support publishing PowerShell modules. But the PowerShell Gallery uses NuGet under the covers. This task creates the module’s .nupkg (`Publish-Module` does that if you give it a file path instead of a URL), which gets saved as an artifact that we can deploy to the gallery if/when we want.